news/2019-04-15-featured-artist-updates-dfs.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Featured Artist Updates: Dictate, Fractal Dreamers & sakuraburst"
+date: 2019-04-15 08:00:00 +0000
+---
+
+We're getting down to earth with some brand new tracks from artists you know and love - namely the Monolithia album from Fractal Dreamers, two new tracks from Dictate, and six songs from sakuraburst's HARPSINGER SINGLES plus more!
+
+[![](/wiki/shared/news/banners/featured-artist.jpg)](https://osu.ppy.sh/beatmaps/artists)
+
+In an ongoing commitment to keep the Featured Artist library as fresh as possible, we've been touching base with many of the artists in the lineup over the past few months, and have snagged some spicy additions to the library from their latest sound.
+
+As with all Featured Artist tracks, these new additions are 100% free for use in mapping and modding within osu!, no ifs or buts.
+
+## Dictate
+
+[![](https://assets.ppy.sh/artists/21/header.jpg)](https://osu.ppy.sh/beatmaps/artists/21)
+
+Featuring *Moth* from [**Dictate**'s](https://osu.ppy.sh/beatmaps/artists/21) contribution to the collab album [MELTDOWN](https://soundcloud.com/renrakuglobalmedia/sets/va-meltdown-free-download) from Renraku and *Secede* from some of his more personal experiments, there's something both familiar and fresh in this bite-sized update.
```

Comment:
```suggestion
Featuring *Moth* from [**Dictate**](https://osu.ppy.sh/beatmaps/artists/21)'s contribution to the collab album [MELTDOWN](https://soundcloud.com/renrakuglobalmedia/sets/va-meltdown-free-download) and *Secede* from some of his more personal experiments, there's something both familiar and fresh in this bite-sized update.
```
removed "from renraku" because I think it's easy to misread this as "from Renraku and Secede" as if they're related, and then the whole sentence makes no sense-- probably a nitpick but this doesn't seem like super important info anyway (?)